sweetest/src/main/java/com/mysugr/sweetest/framework/configuration/ModuleTestingConfiguration.kt

```diff
@@ -31,6 +31,8 @@ fun moduleTestingConfiguration(
     val scope = Dsl.MainScope()
     run?.invoke(scope)
 
+    check(scope.dependencies.isNotEmpty()) { "The module configuration must not be empty!" }
```

Comment:
what is the thought behind this? just thinking: what if someone has just created an empty testing configuration to use it later or there is an empty configuration lying around already? unlikely and edgy but an unnecessary bummer, probably, and strictly speaking a breaking change...

Reply:
fails a test currently 😬

Reply:
What would be the purpose of creating an empty testing configuration? And if there are empty ones lying around they should be cleaned up IMO.

Reply:
as talked about f2f: replace by a `println` deprecation message (pointing towards sweetest 2.0.0 probably...?)